### PR TITLE
Release improvements

### PR DESCRIPTION
**Pull request by tjanez**
Add production `config.js`.
Add instructions on preparing releases to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,15 @@ yarn test
 yarn buildProd
 ```
 
+## Preparing a Release
+
+```sh
+VERSION=<VERSION>
+rm -r dist/
+yarn buildProd
```
Comment (pro-wh) [inline README.md:107]: imo also include the command for installing dependencies

File: README.md [hunk 2]
```diff
@@ -99,6 +99,15 @@ yarn test
 yarn buildProd
 ```
 
+## Preparing a Release
+
+```sh
+VERSION=<VERSION>
+rm -r dist/
```
Comment (pro-wh) [inline README.md:106]: maybe add `-f` in case we later copy this script to CI where we'll maybe start without a dist dir

**Review by pro-wh — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tjanez) [reply README.md]: Agreed. Done.

File: README.md (hunk 1 above)
Comment (tjanez) [reply README.md:108]: Agreed. Done.

File: config.js [hunk 3]
```diff
@@ -10,34 +20,37 @@ export const cointypes = {
 }
 
 /**
- * Lock time, minute
+ * Lock time (in seconds).
  */
-export const LOCK_TIME = 24 * 60
+export const LOCK_TIME = 5 * 60
```
Comment (lukaw3d) [inline config.js:25]: Time is not in seconds. Did you mean to change https://github.com/oasisprotocol/oasis-wallet-ext/blob/1922c520330c94a36fce9b542efd236c43988ea6/src/background/service/APIService.js#L76-L90

Comment (lukaw3d) [reply config.js:25]: Fixed in https://github.com/oasisprotocol/oasis-wallet-ext/pull/133

Comment (tjanez) [reply config.js:25]: Oh, thanks for catching this @lukaw3d :slightly_smiling_face: !

**Pull request merged**